ci/docker.sh

```diff
@@ -14,9 +14,12 @@ github_ref_name="${GITHUB_REF_NAME:-}"
 common_file_location="${COMMON_FILE_LOCATION:-not-set}"
 docker_file_path='docker/dockerfiles/zkv-node.Dockerfile'
 
+private_docker_repo="${PRIVATE_DOCKER_REPO:-}"
```

Comment:
Instead of introducing new VAR, perhaps try to use existing

```
docker_image_build_name="${DOCKER_IMAGE_BUILD_NAME:-zkverify}"
```

Reply:
Do you mean removing the new variable and passing `DOCKER_IMAGE_BUILD_NAME` as `horizenlabs/zkverify-dev`, right?

Reply:
updated